CHANGES.md

```diff
@@ -49,6 +49,7 @@
   - Added new counters in `Stats` (#1570, @Ngoguey42).
   - Added an option to configure the index function and pick the relevant bits
     in a cryptographic hash by default (#1677 #1699, @samoht)
+  - Verify inode depth invariants (#1711, @Ngoguey42). Forward port of #1665.
```

Comment:
I'm not sure what is the best way to add these forward port in the changelog. @CraigFe added both the initial and the forward port PR as one entry in the changelog ([for instance](https://github.com/mirage/irmin/pull/1699/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R51)). We can do this, but then do we keep this entry in the irmin.3.0 changelog? (even though it is included in irmin.2.10 too?)